packages/nextjs/utils/scaffold-eth/decodeTxData.ts

```diff
@@ -34,12 +35,15 @@ export const decodeTransactionData = (tx: TransactionWithFunction) => {
           abi: contractAbi as AbiFunction[],
           name: functionName,
         })?.inputs.map((input: any) => input.type);
-
+        foundInterface = true;
         break;
       } catch (e) {
-        console.error(`Parsing failed: ${e}`);
+        // NOTE: do nothing
       }
     }
+    if (!foundInterface) {
+      tx.functionName = "Unknown ABI";
```

Comment:
Shall we do "Unknown Function" here?

Reply:
I agree, what do you think about ":warning: Unknown"?

Reply:
![Screenshot from 2024-11-20 13-07-53](https://github.com/user-attachments/assets/0ce7206e-74ca-41a1-9a57-9c5cb5b44ff8)

Reply:
Looks good!